### PR TITLE
Light tile remotes

### PR DESCRIPTION
**Pull request by unid15**
- Adds light tile remotes, tools that can modify colours of light tiles at distance
- They're also assemblies so they're fully compatible with assembly frames
- There's a 1 second cooldown when connected to assemblies, and 0.5 second cooldown when used manually

### CODE REVIEW

**Comment by ihadtoregisterforthis** [comment]
Thanks doc.

**Comment by 9600bauds** [comment]
this is insanely neat

**Comment by Intigracy** [comment]
light tile remote?

**Comment by Exxion** [comment]
It appears that you done fucked up with Git

**Comment by FudgePucker** [comment]
11/10
Everything I wanted and more

**Comment by Intigracy** [comment]
ignore the extra commits it's clusterfack's fault.

**Comment by PJB3005** [comment]
:+1: :x: :100:

File: code/modules/assembly/light_tiler.dm [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+	"}
+
+	var/datum/browser/popup = new(user, "\ref[src]", "[src]", 500, 300, src)
+	popup.set_content(dat)
+	popup.open()
+
+	onclose(user, "\ref[src]")
+
+/obj/item/device/assembly/light_tile_control/Topic(href, href_list)
+	if(..()) return
```
Comment (PJB3005) [inline code/modules/assembly/light_tiler.dm]: Should be returning `1` **or** `..()`.

**Comment by Zth--** [comment]
:+1:

**Comment by 9600bauds** [comment]
dose
Future reference for when this gets merged

**Comment by Intigracy** [comment]
Thanks doc.

**Comment by unid15** [comment]
Ready for merge @N3X15

**Pull request merged**